.gitignore

```diff
@@ -38,13 +38,9 @@ temp/
 # Jasmine SpecRunner
 _SpecRunner.html
 
-# Ignore .html files in root
-/*.html
-!fab-toolbar-demo.html
-
```

Comment:
hmm wait, why removing html files from .gitignore? Shouldn't this be ignoring html files inside `docs` folder?

Reply:
The issue with that is that the files can't then be published to gh-pages as they're in the gitignore. The script could remove the exceptions so that the build can be pushed but that feels like a bodge.

Reply:
hmm do the `peaceiris/actions-gh-pages` action push the new commit of compiled files to the `v1-dev`?

Reply:
No, they push to gh-pages but use the gitignore (https://github.com/peaceiris/actions-gh-pages/issues/445)

Reply:
The script now removes the gitignore rule when building the docs

Reply:
hmm but the `docs` folder don't have any `.gitignore`. How this is relate to the issue you mentioned? 💭

Reply:
Hmm, I think there must be a bug that makes it use the root .gitignore as changing that fixes the issue